locustempus/main/viewsets.py

```diff
@@ -30,3 +32,30 @@ class EventApiView(ModelViewSet):
     """Retrieves events"""
     serializer_class = EventSerializer
     queryset = Event.objects.all()
+
+
+class ResponseApiView(ModelViewSet):
+    """Retrieves responses"""
+    serializer_class = ResponseSerializer
+
+    def get_queryset(self):
+        qs = Response.objects.all()
```

Comment:
Question - should this default to `Response.objects.none()` rather than `.all()`? I'm wondering about permissions here.

Reply:
Good point. I refactored this to take into account what a user's role is in the course that the activity belongs to.